test-core.ini

```diff
@@ -16,11 +16,11 @@ testing = true
 
 
 # Specify the Postgres database for SQLAlchemy to use
-sqlalchemy.url = postgresql://ckan_default:pass@localhost/ckan_test
+sqlalchemy.url = postgresql://ckan_default:ckan@localhost/ckan_test
```

Comment:
Please don't change the defaults on this file as it will affect existing scripts